src/System.IO.UnmanagedMemoryStream/tests/UmaTests.cs

```diff
@@ -23,6 +23,13 @@ public static void UmaInvalidReadWrite()
                 Assert.Throws<ArgumentOutOfRangeException>(() => uma.Write(capacity, 12345));
                 Assert.Throws<ArgumentException>(() => uma.Write(capacity - 1, 0.123));
 
+                // UMA should throw when reading bad decimal values (some bits of flags are reserved and must be 0)
+                uma.Write(0, 0); // lo
```

Comment:
[They did, actually](http://i.imgur.com/WhD0Us8.png)-- I don't know why they're failing with CI. Also, I checked line 838 (where Jenkins said the null ref happened) and there's no way an exception could have been thrown there:

```csharp
*((Int32*)pointer) = value;
```

I don't know, perhaps it's a problem with CI?